client/signup/main.jsx

```diff
@@ -536,7 +536,6 @@ class Signup extends React.Component {
 								hideFreePlan={ hideFreePlan }
 								{ ...propsFromConfig }
 							/>
-							{ this.props.shouldShowMockups && <SiteMockups stepName={ this.props.stepName } /> }
 						</>
```

Comment:
This fragment can be removed now. Just means we can trim off one level of indentation :)